fuzz/fuzz_targets/hfuzz.rs

```diff
@@ -1,17 +1,20 @@
+extern crate codespan;
 #[macro_use]
 extern crate honggfuzz;
 extern crate rcc;
 
+use codespan::FileId;
+use rcc::{Files, Locatable};
 use rcc::data::lex::{Keyword::*, Token};
-use rcc::Locatable;
 use rcc::PreProcessor;
+use std::default::Default;
 
-fn lex(s: &str) -> PreProcessor {
-    PreProcessor::new("<test-suite>", s.chars(), false)
+fn lex<'s>(s: &'s str, file: FileId, files: &'s mut Files) -> PreProcessor<'s> {
+    PreProcessor::new(file, s, false, files)
```

Comment:
Honestly at this point it probably makes sense to get rid of `lex()` altogether, we're writing way more code then we're saving

Reply:
done